libdevcore/SHA3.cpp

```diff
@@ -192,6 +199,14 @@ static inline int hash(uint8_t* out, size_t outlen,
 	if (outlen > (bits/8)) {                                      \
 	  return -1;                                                  \
 	}                                                             \
+	return hash(out, outlen, in, inlen, 200 - (bits / 4), 0x06);  \
```

Comment:
I'm trying to understand this `0x06` and searching for "Keccack SHA3 difference" online, but so far I haven't gotten a clue.